pkg/odo/cli/component/delete.go

```diff
@@ -28,10 +28,24 @@ import (
 // DeleteRecommendedCommandName is the recommended delete command name
 const DeleteRecommendedCommandName = "delete"
 
-var deleteExample = ktemplates.Examples(`  # Delete component named 'frontend'. 
+var deleteExample = ktemplates.Examples(`
+# Delete the component present in the current directory
+%[1]s
+
+# Delete the component named 'frontend'
 %[1]s frontend
-%[1]s frontend --all
-  `)
+
+# Delete the component and all the related config files from the current directory
+%[1]s --all
+
+# Delete the component present in the './frontend' directory
```

Comment:
```suggestion
# Delete the component present in the './frontend' directory from the cluster
```
Because it deletes the resources created in devfile.yaml from the cluster. It's actually not sufficient to say just "delete the component" because the `ownerRefereneces` also ensures that its URL, storage, Operator backed service and link also get deleted if they exist.